### PR TITLE
Functional  tests - Add test merchandise return options

### PR DESCRIPTION
**Pull request by nesrineabdmouleh**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | 1.7.8.x
| Description?      | Add test merchandise return options
| Type?             | refacto
| Category?         | TE
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | no
| How to test?      | TEST_PATH="functional/BO/05_customerService/03_merchandiseReturns/01_merchandiseReturnOptions.js" URL_FO=yourShopURL npm run specific-test
| Possible impacts? | no

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/24652)

### CODE REVIEW

**Review by NeOMakinG — DISMISSED** [review]

File: tests/UI/pages/BO/customerService/merchandiseReturns/index.js [hunk 1]
```diff
@@ -39,6 +40,18 @@ class MerchandiseReturns extends BOBasePage {
     await this.clickAndWaitForNavigation(page, this.saveButton);
     return this.getTextContent(page, this.alertSuccessBlock);
   }
+
+  /**
+   * Update returns prefix
+   * @param page
+   * @param prefix
```
Comment (Progi1984) [inline tests/UI/pages/BO/customerService/merchandiseReturns/index.js]: Please set type

**Review by Progi1984 — COMMENTED** [review]
A small JSDoc and it's ok

File: tests/UI/pages/BO/customerService/merchandiseReturns/index.js (hunk 1 above)
Comment (nesrineabdmouleh) [reply tests/UI/pages/BO/customerService/merchandiseReturns/index.js]: Done thx :+1:

**Review by NeOMakinG — APPROVED** [review]

**Review by Progi1984 — APPROVED** [review]

**Comment by Progi1984** [comment]
Thanks @nesrineabdmouleh

**Pull request merged**